frontend/src/functions/formatPhone.test.ts

```diff
@@ -0,0 +1,13 @@
+import { formatPhone } from "./formatPhone";
+
+it("returns a formatted number even if not a valid length", () => {
+  const phoneNumber = "+122244";
+
+  expect(formatPhone(phoneNumber)).toStrictEqual("+1 (222) 44-");
```

Comment:
[`toStrictEqual`](https://jestjs.io/docs/expect#tostrictequalvalue) is intended to compare objects, making sure that both sides of the comparison have the same properties. Since this is just a string, [`toBe`](https://jestjs.io/docs/expect#tobevalue) should be sufficient. It's not a biggie in a single case (it's mostly a performance concern, to avoid our unit tests from running too long), but it's good to get in the habit of using the right matchers in the right case, especially since people might copy-paste your unit tests to base their own future tests on :)

```suggestion
  expect(formatPhone(phoneNumber)).toBe("+1 (222) 44-");
```

(Also for the other test below.)